core/reflection.jvm/resources/META-INF/com.android.tools/proguard/kotlin-reflect.pro

```diff
@@ -0,0 +1,23 @@
+# When editing this file, update the following files as well:
+# - META-INF/com.android.tools/r8-from-1.6.0/kotlin-reflect.pro
+# - META-INF/com.android.tools/r8-upto-1.6.0/kotlin-reflect.pro
+# - META-INF/proguard/kotlin-reflect.pro
+# Keep Metadata annotations so they can be parsed at runtime.
+-keep class kotlin.Metadata { *; }
+
+# Keep implementations of service loaded interfaces
+# R8 will automatically handle these these in 1.6+
```

Comment:
Nit: this file is not quite related to `R8`. :) (and below too)

Reply:
D'oh, fixed in 5091302